src/pages/content/word-list.mdx

```diff
@@ -1,12 +1,707 @@
+  <Do>
+    <p>It lists all your customers.</p>
+  </Do>
+  <Dont>
+    <p>It lists all of your customers.</p>
+  </Dont>
+</BestPractice>
+
+### among, between
+
+Use among when referring to 3 or more persons (or things).
+
+- Compare with [between](/content/word-list#between)
+
+<BestPractice>
+  <Do>
+    <p>You can share tickets among members of your group.</p>
+  </Do>
+  <Dont>
+    <p>You can share tickets among you and your admin.</p>
+  </Dont>
+</BestPractice>
+
+### and/or
+
+Avoid this construction. Use either and or or by itself.
+
+### Answer Bot
+
+Two words, title case.
+
+### any time, anytime
+
+The two-word any time is always the safer choice. If you do use the one-word
+form, anytime, make sure it's an adverb. If you need to say, “At any time,” it
+must be two words.
+
+### app, apps
+
+Use lowercase for general references. Don’t capitalize for emphasis. Capitalize
+only when referring to a feature, such as Zendesk Apps. When referring to the
+Zendesk app marketplace use Zendesk Marketplace.
+
+### appears, displays
+
+Displays is a transitive verb, meaning that the action transfers to something
+else. For example, a screen displays an image. While displays requires a direct
+object, appears does not. For example, an image appears on screen.
+
+### as
+
+Avoid using as as a synonym for because or while. While grammatically correct,
+these uses make reading more difficult for a global audience.
+
+- Compare with [because](/content/word-list#because)
+
+<BestPractice>
+  <Do>
+    <p>Use the active voice whenever possible because it’s easier to translate.</p>
+  </Do>
+  <Dont>
+    <p>Use the active voice whenever possible as it is easier to translate.</p>
+  </Dont>
+</BestPractice>
+
+### because, since
+
+Use because instead of since. The reason is that since can cause problems with
+machine translation. Use because to refer to the reason for something, and since
+to refer to a passage of time.
+
+### between
+
+Use between when referring to 2 persons (or things) or when referring to
+relationships between two items at a time, regardless of the total number of
+items.
+
+- Compare with [among](/content/word-list#among)
+
+<BestPractice>
+  <Do>
+    <p>Move between the two tickets at the top of the list.</p>
+  </Do>
+  <Dont>
+    <p>Move between several tickets on the page.</p>
+  </Dont>
+</BestPractice>
+
+### blacklist, whitelist
+
+Avoid this usage. Use blocked or allowed instead.
+
+### CC, CCs, CCed
+
+In the product, use uppercase. OK to use as a noun ("the CC", "the CCs"), as a
+transitive verb ("CC me"), or as an adjective ("the CCed agent").
+
+In documentation, it’s permitted to use lowercase and apostrophes (cc, cc’s,
+cc’ed).
+
+### channel
+
+Use lowercase for general references. Capitalize when referring to t`he Channel
+framework or describing features of the framework, such as the Channel pull
+endpoint.
+
+### chat, Zendesk Chat
+
+Lowercase when talking about a chat message or the verb. Capitalize when
```

Comment:
Lowercase when talking about a chat message or the verb. Capitalize when
referring to the feature or the product, such as _Zendesk Chat_.